build/tasks/test-on-node.js

```diff
@@ -80,7 +80,10 @@ module.exports = function( grunt ) {
 
 		QUnit.config.autostart = false;
 
-		require( "../../" + file );
+		var filePath = path.resolve( __dirname, "../../" + file + ".js" );
```

Comment:
Nice catch! But I think we want `var filePath = require.resolve( "../../" + file )`, and maybe even a generalized `requireFresh`.